internal/common/labels.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+import "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+const (
+	AppKubernetesNameLabel      = "app.kubernetes.io/name"
+	AppKubernetesPartOfLabel    = "app.kubernetes.io/part-of"
+	AppKubernetesVersionLabel   = "app.kubernetes.io/version"
+	AppKubernetesManagedByLabel = "app.kubernetes.io/managed-by"
+	AppKubernetesComponentLabel = "app.kubernetes.io/component"
+)
+
+type AppComponent string
+
+func (a AppComponent) String() string {
+	return string(a)
+}
+
+const (
+	AppComponentMonitoring AppComponent = "monitoring"
+	AppComponentSchedule   AppComponent = "schedule"
+	AppComponentTemplating AppComponent = "templating"
```

Comment:
This is the component I came up with for templating stuff :man_shrugging: 
Feel free to suggest a different component for `template-validator` and `common-templates`
Here is the list the HCO folks defined: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/994/files#diff-65ef3f8820df1f74c24a4326ce70629156558895a1d1afff1f8f857cdbc24638R56